src/sourmash/__init__.py

```diff
@@ -16,6 +16,7 @@ class MinHash - hash sketch class
 BSD 3-Clause license.
 """
 from deprecation import deprecated
+from pkg_resources import get_distribution
```

Comment:
```suggestion
from importlib.metadata import version
```

(maybe add fallbacks for py3.8 too)